tests/API.spec.js

```diff
@@ -106,7 +106,7 @@ describe( "API with Humps", function ()
         expect( flag ).toBe( "Success" );
     } );
 
-    it( "should transform requests", function ()
+    it( "should transform post requests", function ()
```

Comment:
Do you mean post like "after" or post like HTTP POST tho, the capitalization is vaguely confusing.  Particularly since queryUrl is used with get, not post.